docs/resources/project_approval_rule.md

```diff
@@ -1,10 +1,8 @@
 # gitlab\_project\_approval\_rule
 
-This resource allows you to create and manage multiple approval rules for your GitLab
-projects. For further information on approval rules, consult the [gitlab
-documentation](https://docs.gitlab.com/ee/api/merge_request_approvals.html#project-level-mr-approvals).
+This resource allows you to create and manage multiple approval rules for your GitLab projects. For further information on approval rules, consult the [gitlab documentation](https://docs.gitlab.com/ee/api/merge_request_approvals.html#project-level-mr-approvals).
 
--> This feature requires a GitLab Starter account or above.
+> **NOTE**: This feature requires a GitLab Starter account or above.
```

Comment:
What's the reason for the NOTE changes? By the way these symbols `->` are callouts.

See: https://www.terraform.io/docs/registry/providers/docs.html#callouts
See: https://registry.terraform.io/providers/gitlabhq/gitlab/latest/docs/resources/project_approval_rule

You can use this tool to test out docs formatting changes: https://registry.terraform.io/tools/doc-preview

Reply:
Aha! Thanks @armsnyder, didn't know about these callouts. My Markdown render doesn't know either. I reverted that line.